readme.md

```diff
@@ -39,6 +40,40 @@ The Open Effects Association (OFX), a non-profit organization, develops and prom
 
 The Association's initial focus is to improve the OpenFX image processing plug-in standard. This goal of this standard is to reduce development effort and support needed for plug-ins across different compositing and editing host platforms.
 
+# Building
```

Comment:
You could refer to `scripts/build-cmake.sh` instead of including the details here.